test/app/model/Knight.spec.ts

```diff
@@ -0,0 +1,195 @@
+import { Knight } from '../../../src/app/model/Knight';
+import { Square } from '../../../src/app/model/Types';
+import 'jest-extended';
+import { boardWithPieces, emptyBoard } from './BoardFixture';
+
+describe('Knight movement', () => {
```

Comment:
Nooo pięknie testy! :)  Jedynie co to ten poczatek bym skasowal z kazdego:
`Check possible squares to go` i zamienil `Knight movement` na `Knight possible moves`